### PR TITLE
equesting a pull to mozilla:master from k88hudson:t2248

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @dseif received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/6c0254d5a32ad7f/output.txt

File: cornfield/app.js [hunk 1]
```diff
@@ -258,7 +258,8 @@ app.post( '/api/publish/:id',
       }
 
       // This is a query string-only URL because of the <base> tag
-      var remixUrl = "?savedDataUrl=/api/remix/" + project._id;
+      var remixUrl = "?savedDataUrl=/api/remix/" + project._id,
+          attribURL = Array.isArray( projectData.media[ 0 ].url ) ? projectData.media[ 0 ].url[ 0 ] : projectData.media[ 0 ].url;
```
Comment (dseif) [inline cornfield/app.js]: storeprojectData.media[ 0 ] just so this line isn't so long

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/6c0254d5a32ad7f/output.txt

Total script time: 3.00 mins

File: css/embed.less [hunk 2]
```diff
@@ -96,99 +98,205 @@ body {
...
-    line-height: 1em;
-    text-decoration: none;
+  &:hover {
+    opacity: 1;
+    .attribution-text {
+      visibility: visible;
+    }
   }
+}
+.attribution-text { // "Credits"
```
Comment (dseif) [inline css/embed.less]: move comment plz

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @dseif received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/7897ce7c7b5cc84/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/7897ce7c7b5cc84/output.txt

Total script time: 3.00 mins

**Pull request closed without merging**